packages/migrate/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "webpack-cli-migrate",
+    "version":"2.0.12",
+    "description": "CLI for webpack & friends",
+    "license": "MIT",
+    "private": true,
+    "engines": {
+      "node": ">=6.11.5"
+    },
+    "dependencies": {
+      "chalk": "^2.3.2",
+      "cross-spawn": "^6.0.5",
+      "global-modules": "^1.0.0",
+      "got": "^8.2.0",
+      "jscodeshift": "^0.5.0",
+      "lodash": "^4.17.5",
+      "p-each-series": "^1.0.0",
+      "p-lazy": "^1.0.0",
+      "prettier": "^1.5.3",
+      "webpack-cli-init": "2.0.12",
+      "webpack-cli-utils": "2.0.12",
+      "yeoman-environment": "^2.0.0",
+      "yeoman-generator": "^2.0.3"
+    }
```

Comment:
There seems to be a lot of extraneous dependencies everywhere like `cross-spawn`, `global-modules`, `got`, `yeoman-environment` which can be cleaned.

`prettier` is part of `utils` - `runPrettier`. Shouldn't be included here.
We don't use `yeoman-generator` as well in migrate.